pkg/testutil/testutil.go

```diff
@@ -58,3 +59,24 @@ func tryAllocTestURL() string {
 	testAddrMap[addr] = struct{}{}
 	return addr
 }
+
+const (
+	waitMaxRetry   = 100
+	waitRetrySleep = time.Millisecond * 50
+)
+
+// CheckFunc is a condition checker that passed to WaitUtil. Its implementation
+// may call c.Fatal() to abort the test, or c.Log() to add more information.
+type CheckFunc func(c *check.C) bool
+
+// WaitUtil repeatly evaluates f() for a period of time, util it returns true.
+func WaitUtil(c *check.C, f CheckFunc) {
```

Comment:
do we need to supply another `WaitUtilWIth(c *check.C, f CheckFunc, retryCount int, waitSleep Duration)` to let outer use customized retry count and sleep time?